common/src/main/resources/assets/goofyplugin/lang/en_us.json

```
@@ -23,6 +23,7 @@
   "figura.docs.goofy.is_error_enabled": "Checks if error logging is enabled",
   "figura.docs.goofy.is_trace_enabled": "Checks if trace logging is enabled",
   "figura.docs.goofy.load_avatar": "Load the avatar of a player based on their UUID\nGN approved",
+  "figura.docs.goofy.get_player_color": "Get the color of a player's avatar from their UUID",
```

and this, plus mention hex
```suggestion
  "figura.docs.goofy.get_avatar_color": "Get the color (in hex) of a player's avatar from their UUID",
```